code/game/gamemodes/objective_items.dm

```diff
@@ -179,7 +179,7 @@
 	return add_item_to_steal(src, /obj/item/clothing/accessory/lawyers_badge)
 
 /datum/objective_item/steal/traitor/chief_engineer_belt
-	name = "the chief engineer's belt"
+	name = "the site foreman's belt"
```

Comment:
```suggestion
	name = "the Site Foreman's belt"
```
Do we want all occurrences like this to be proper nouns like this 

`name = "\improper Site Foreman's Office"`